frontend/src/openapi/models/sdkContextSchema.ts

```diff
@@ -26,5 +26,5 @@ export interface SdkContextSchema {
     sessionId?: string;
     /** An identifier for the current user */
     userId?: string;
-    [key: string]: any;
+    [key: string]: string;
```

Comment:
you have to fix it manually because we added a new backend type that breaks orval